src/components/Form/Form.tsx

```diff
@@ -8,16 +9,26 @@ interface IState{
   } | undefined
 }
 
-const Form:React.FC <IState> = ({ questions }) => {
+const Form:React.FC <IState> = ({ questions, currentCountry}) => {
+  const [answer,setAnswer] = useState<string | number>('')
+
+  const handleSubmit = (e:any) => {
```

Comment:
We'll have to look into what type an event is. For now though, having it set as any will let us figure out the functionality.